_posts/wookiemage/2016-1-18-Chapter-Three-Exercises.md

```diff
@@ -0,0 +1,15 @@
+---
+author: wookiemage
+layout: post
+title: Chapter Three: Flow Control Exercises
+---
+Here's my first exercise:
+<iframe src="https://trinket.io/embed/python/01b02c1c22" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Here's the second one:
+<iframe src="https://trinket.io/embed/python/04aea8e8eb" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Here's the final one:
+<iframe src="https://trinket.io/embed/python/90f2e61a9a" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+I bet there is a more efficient way of doing this last one. I ended up using both a "try" and an "if". I think that if I knew a little bit more about data types and how they are evaluated in Python I could have just used one or the other.
```

Comment:
That's actually a pretty efficient way of doing it using just if statements.  The most computationally efficient way of doing it might be a dictionary lookup of some sort, which we'll get to later in the semester.